tsl/test/expected/cagg_api.out

```diff
@@ -0,0 +1,159 @@
+ hypertable_id | schema_name | table_name | created 
+---------------+-------------+------------+---------
+             2 | public      | hyper_tstz | t
+(1 row)
+
+SELECT * FROM create_hypertable('hyper_no_cagg', 'time');
+ hypertable_id | schema_name |  table_name   | created 
+---------------+-------------+---------------+---------
+             3 | public      | hyper_no_cagg | t
+(1 row)
+
+SELECT * FROM create_hypertable('hyper_multi', 'time', 'device', 4);
+ hypertable_id | schema_name | table_name  | created 
+---------------+-------------+-------------+---------
+             4 | public      | hyper_multi | t
+(1 row)
+
+INSERT INTO hyper_ts
+  SELECT time, ceil(random() * 100)::int
+    FROM generate_series('2025-01-01'::timestamp,
+                         '2025-01-06', '1m') time;
+INSERT INTO hyper_tstz
+  SELECT time, ceil(random() * 100)::int
+    FROM generate_series('2025-01-01'::timestamptz,
+                         '2025-01-06', '1m') time;
+CREATE MATERIALIZED VIEW ts_temperature_1h
+  WITH  (timescaledb.continuous) AS
+SELECT time_bucket('1 hour', time), avg(value)
+  FROM hyper_ts
+GROUP BY 1;
+NOTICE:  refreshing continuous aggregate "ts_temperature_1h"
+CREATE MATERIALIZED VIEW ts_temperature_15m
+  WITH  (timescaledb.continuous) AS
+SELECT time_bucket('15 minutes', time), avg(value)
+  FROM hyper_ts
+GROUP BY 1;
+NOTICE:  refreshing continuous aggregate "ts_temperature_15m"
+CREATE MATERIALIZED VIEW tstz_temperature_1h
+  WITH  (timescaledb.continuous) AS
+SELECT time_bucket('1 hour', time), avg(value)
+  FROM hyper_tstz
+GROUP BY 1;
+NOTICE:  refreshing continuous aggregate "tstz_temperature_1h"
+CREATE MATERIALIZED VIEW tstz_temperature_15m
+  WITH  (timescaledb.continuous) AS
+SELECT time_bucket('15 minutes', time), avg(value)
+  FROM hyper_tstz
+GROUP BY 1;
+NOTICE:  refreshing continuous aggregate "tstz_temperature_15m"
+CREATE MATERIALIZED VIEW multi_temperature_15m
+  WITH  (timescaledb.continuous) AS
+SELECT time_bucket('15 minutes', time), avg(value)
+  FROM hyper_multi
+GROUP BY 1;
+NOTICE:  continuous aggregate "multi_temperature_15m" is already up-to-date
+SET search_path TO _timescaledb_functions, public;
+-- These are not part of the API, but we test them here just to make
+-- sure they work as expected.
+SELECT table_name, get_materialization_info(table_name)
+  FROM (
+      VALUES ('tstz_temperature_15m'), ('multi_temperature_15m')
+  ) t(table_name);
+      table_name       | get_materialization_info 
+-----------------------+--------------------------
+ tstz_temperature_15m  | (8,"@ 15 mins")
+ multi_temperature_15m | (9,"@ 15 mins")
+(2 rows)
+
+\set ON_ERROR_STOP 0
+SELECT get_materialization_info('hyper_no_cagg');
+ERROR:  "public.hyper_no_cagg" is not a continuous aggregate
+\set ON_ERROR_STOP 1
+SELECT * INTO before FROM _timescaledb_catalog.continuous_aggs_materialization_invalidation_log;
+CALL _timescaledb_functions.add_materialization_invalidations(
+     'tstz_temperature_15m'::regclass,
```

Comment:
Why do you need argument casts here?

Reply:
It is needed for the second argument, so just adding it here as well. If you feel strongly about it, I can remove it.